### PR TITLE
readme-enho

### LINKED ISSUE #196: readme-enho

### PR DESCRIPTION
**Pull request by albertmink**
Fixes #196

### CODE REVIEW

File: file-formats/enho/README.md [hunk 1]
```diff
@@ -1,9 +1,6 @@
...
-For ENHO (BAdI Enhancement Implementation), there is a metadata file.
-The JSON schema for the ENHO metadata file is provided [here](./enho.json).
 
-Example files for an ENHO `z_aff_example_enho` are provided in the following table.
-File | Content
- --- | ---
-[`z_aff_example_enho.enhs.json`](./examples/z_aff_example_enho.enho.badi.json)    | Metadata of the ENHO
+File | Cardinality | Definition | Schema | Example
+:--- | :---  | :--- | :--- | :---
+`<name>.enho.json` | 1 | [`zif_aff_enho_v1.intf.abap`](./type/zif_aff_enho_v1.intf.abap) | [`enho.json`](./enho.json) | [`z_aff_example_enho.enhs.json`](./examples/z_aff_example_enho.enho.badi.json)
```
Comment (wurzka) [inline file-formats/enho/README.md:6]:
```suggestion
`<name>.enho.json` | 1 | [`zif_aff_enho_v1.intf.abap`](./type/zif_aff_enho_v1.intf.abap) | [`enho.json`](./enho.json) | [`z_aff_example_enho.enho.json`](./examples/z_aff_example_enho.enho.badi.json)
```

**Review by wurzka — CHANGES_REQUESTED** [review]
I found just one typo (which has been there before...)

File: file-formats/enho/README.md [hunk 2]
```diff
@@ -1,9 +1,6 @@
...
-For ENHO (BAdI Enhancement Implementation), there is a metadata file.
-The JSON schema for the ENHO metadata file is provided [here](./enho.json).
 
-Example files for an ENHO `z_aff_example_enho` are provided in the following table.
-File | Content
- --- | ---
-[`z_aff_example_enho.enhs.json`](./examples/z_aff_example_enho.enho.badi.json)    | Metadata of the ENHO
+File | Cardinality | Definition | Schema | Example
+:--- | :---  | :--- | :--- | :---
+`<name>.enho.json` | 1 | [`zif_aff_enho_v1.intf.abap`](./type/zif_aff_enho_v1.intf.abap) | [`enho.json`](./enho.json) | [`z_aff_example_enho.enho.json`](./examples/z_aff_example_enho.enho.badi.json)
```
Comment (wurzka) [inline file-formats/enho/README.md:6]:
```suggestion
`<name>.enho.json` | 1 | [`zif_aff_enho_v1.intf.abap`](./type/zif_aff_enho_v1.intf.abap) | [`enho.json`](./enho.json) | [`z_aff_example_enho.enho.badi.json`](./examples/z_aff_example_enho.enho.badi.json)
```

**Review by wurzka — CHANGES_REQUESTED** [review]
Sorry, just one more thing

File: file-formats/enho/README.md (hunk 2 above)
Comment (albertmink) [reply file-formats/enho/README.md:6]: But what about the `<name>.enho.json`. Is it still valid?

Comment (wurzka) [reply file-formats/enho/README.md:6]: Your are right. According to the example, this has to be `<name>.enho.badi.json`

**Review by wurzka — APPROVED** [review]

**Pull request merged**